src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -147,4 +147,9 @@
     <PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="$(AspNetCoreVersion)" PrivateAssets="None" Condition="'$(BUILD_PACKAGE_CACHE)' == 'true'" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="build\**\*.xml" PackagePath="%(Identity)" />
```

Comment:
I will be adding the manifests. This has to be done in Coherence-PackageCache though since that's when we actually generate the runtime store.